superset/migrations/versions/8786d6374caa_add_column_for_query_estimate.py

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is there a reason we wouldn't want to enable this for all supported dbs if the feature flag is turned on? It seems like the db migration might be overkill here. More of a design question tho.

Reply:
I think it might be possible to have users running different versions of Presto (prod and staging, eg), so enabling the feature per DB seems to make sense.

Reply:
As mentioned above, maybe enable per DB through using the extra data version instead?

Reply:
Will do.